src/vs/workbench/contrib/notebook/browser/controller/editActions.ts

```diff
@@ -621,3 +625,62 @@ registerAction2(class SelectNotebookIndentation extends NotebookAction {
 		return;
 	}
 });
+
+registerAction2(class CommentSelectedCellsAction extends NotebookMultiCellAction {
+	constructor() {
+		super({
+			id: COMMENT_SELECTED_CELLS_ID,
+			title: localize('commentSelectedCells', "Comment Selected Cells"),
+			keybinding: {
+				when: ContextKeyExpr.and(
+					NOTEBOOK_EDITOR_FOCUSED,
+					NOTEBOOK_CELL_LIST_FOCUSED,
+					ContextKeyExpr.not(InputFocusedContextKey),
+					NOTEBOOK_EDITOR_EDITABLE,
+					NOTEBOOK_CELL_EDITABLE
+				),
+				primary: KeyMod.CtrlCmd | KeyCode.Slash,
+				weight: KeybindingWeight.WorkbenchContrib
+			}
+		});
+	}
+
+	async runWithContext(accessor: ServicesAccessor, context: INotebookCommandContext): Promise<void> {
+		const languageConfigurationService = accessor.get(ILanguageConfigurationService);
+
+		const selectedCellEditors: ICodeEditor[] = [];
+		context.selectedCells.forEach(cell => {
+			const findContext = { notebookEditor: context.notebookEditor, cell };
+			const foundEditor = findTargetCellEditor(findContext, cell);
+			if (foundEditor) {
+				selectedCellEditors.push(foundEditor);
+			}
+		});
+
+		selectedCellEditors.forEach(editor => {
+			if (!editor.hasModel()) {
+				return;
+			}
+
+			const model = editor.getModel();
+			const commands: ICommand[] = [];
+			const modelOptions = model.getOptions();
+			const commentsOptions = editor.getOption(EditorOption.comments);
+
+			commands.push(new LineCommentCommand(
```

Comment:
Our scenario is simpler, I wonder if we can move off the editor command so it can work for cells out of viewport.